docs/development/How-To-User-Interface.md

```diff
@@ -16,11 +18,11 @@ Make sure to attach the relevant GUI Script component to the prefab itself (In t
 
 ### GUI Script Tips
 
-To be expanded. Include Netpage assignment variable declaration, ultisation of UpdateDisplay(), declartion in NetTab.cs for NetTabType enumerical  and communication to backend script.
+To be expanded. Include Netpage assignment variable declaration, ultilisation of UpdateDisplay(), declaration in NetTab.cs for NetTabType enumerical  and communication to backend script.
```

Comment:
is it supposed to be utilisation?

Reply:
Yes, sorry.

Reply:
I don’t know how to approve the changes on my phone, but your correction is right.